### PR TITLE
Increase component test coverage for Wallet

### PR DESCRIPTION
**Pull request by arpadt**
https://bullioncapital.atlassian.net/browse/ABX-4428

- wrote unit test to most components
- refactored some calls to SweetAlert to make it more testable

### CODE REVIEW

File: src/components/TransferForm.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as copy from 'copy-to-clipboard'
 import * as React from 'react'
 
 import { TransferProps } from '@containers/TransferForm'
-import { formAlert } from '@helpers/alert'
+import { formAlert } from '@helpers/alert';
```
Comment (desnor) [inline src/components/TransferForm.tsx]: this should be removed by linting I would have thought? would be nice to keep the file from having any changes

Comment (arpadt) [reply src/components/TransferForm.tsx]: the linter didn't pick it up, so I removed it.

**Comment by arpadt** [comment]
Added the missing test.

I agree that it might seem to be much and verbose, especially that the service will soon be refactored. I like thorough testing though and I would leave it as is for now. We can always change it when getting to the refactor.

File: src/containers/Sign.ts [hunk 2]
```diff
@@ -40,6 +41,7 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   },
   signMessage: (signature: string) => dispatch(signMessage(signature)),
   messageVerificationResult: (isValid: boolean) => dispatch(messageVerificationResult(isValid)),
+  callFormAlert: (keyAndMessage: FormAlert) => dispatch(invalidForm(keyAndMessage))
```
[the remark this answers is not in the document]
Comment (arpadt) [reply src/containers/Sign.ts]: fixed

**Pull request merged**